Add first draft of designer take home project

andrewsardone opened the pull request:
This drops in a first draft of our "event scheduling" take-home project. A **few questions before merge**:

- Should we duplicate our three intro questions (hardest problem you've recently solved, a recent book or blog post you've enjoyed, and a summary of Nutshell customers according to our website)?
- Should we restructure the repo? Link out to our developer and designer questions from `README.md`?

andrewsardone committed:
Add first draft of designer take home project

andyfowler commented:
> Should we duplicate our three intro questions (hardest problem you've recently solved, a recent book or blog post you've enjoyed, and a summary of Nutshell customers according to our website)?

I think we should keep those for all candidates — we're looking for passion, writing ability, and business acumen. I think they're relevant for anybody who applies.

> Should we restructure the repo? Link out to our developer and designer questions from README.md?

Yeah, I like that idea. Not necessary if you want to get this merged now, but I had originally hoped to spice up the `README` with an image or two, which would make a nice index to the role-specific docs.

andrewsardone committed:
Extract dev & designer questions into separate files

This keeps our common written questions organized in one place (D.R.Y.),
while keeping the visual design and coding questions separated.

andrewsardone commented:
@andyfowler: I've [taken a stab](https://github.com/nutshellcrm/join-the-team/pull/1/commits/325b0e6d2a9fce8777e8c9b27b4181a2258cbde5) at reworking our root `README.md` and separating out the developer and designer questions. Can you take a look?

- [Rendered `README.md`](https://github.com/nutshellcrm/join-the-team/blob/325b0e6d2a9fce8777e8c9b27b4181a2258cbde5/README.md)

andyfowler commented:
@andrewsardone Looks great! 🚢
